src/domain/classify_detections.py

```diff
@@ -80,7 +81,7 @@
 
```

Comment:
Add blank line

Reply:
I guess the comment is to add additional blankline before class definition..

Reply:
Exactly! :)